Frontend/src/Pages/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import Header from "../../components/Dashboard/Header/Header.jsx";
+
+const Dashboard = () => {
+    return (
+      <div className="">
+          <Header />
```

Comment:
No need to include the header on the page since it is already added by the Layout.

Reply:
Removed